proposals/Exceptions.md

```diff
@@ -736,18 +736,24 @@ the event indices (Used for both imports and module-defined).
 
 ## Comparisons of the two proposals
 
-- Proposal 2 is more expressive and possibly provides more flexibility
-  for future frontend developers for other languages.
-
-- As Proposal 2 introduces first-class exception reference type, we have to
-  manage lifetime of exception objects, from which arises several questions.
+- Proposal 2 introduces a first-class exception reference type. This raises several
+  questions about exception lifetime management:
   - How do we manage exception objects' lifetime in non-GC embeddings? Do we
     make reference counting mandatory?
   - Who is responsible for deleting exception objects?
   - What should we do for except_ref values of invalid exception objects already
     deleted?
   - How should exception reference type be related to the existing reference
     type or GC proposal?
+   
+  Consequently, Proposal 1 would be simpler to implement for VMs which do not need
```

Comment:
Hm, this seems to assume that reference types will be an optional feature for future Wasm VMs. I don't think that is going to be possible (even though GC types can be optional).

Reply:
Not possible for future VMs not to implement reference types? If you don't want GC types (and maybe host bindings), why would you need reference types? Without exception references, you could just have MVP + exceptions, no?